### PR TITLE
rewrite to work with swift3

### PR DESCRIPTION
**Pull request by mamdouhweb**
Changed files for the library to work with Swift3

### CODE REVIEW

File: Pod/Classes/WavesLayer.swift [hunk 1]
```diff
@@ -114,56 +84,30 @@ class WavesLayer: CALayer {
...
-            displayAnimation.duration = 0.3
-            displayAnimation.beginTime = CACurrentMediaTime() + Double(num) * 0.3
-            displayAnimation.fromValue = 0.0
-            displayAnimation.toValue = 1.0
-            displayAnimation.removedOnCompletion = false
-            displayAnimation.fillMode = kCAFillModeForwards
-            
-            gradient.addAnimation(displayAnimation, forKey: "opacity")
-            
-            gradient.opacity = 0.0
```
Comment (jakubknejzlik) [inline Pod/Classes/WavesLayer.swift]: Is removing all animations intentional?

Comment (mamdouhweb) [reply Pod/Classes/WavesLayer.swift]: Oh, this was taken from 2.0.1 not from master... it needs to be resolved with master

Comment (mamdouhweb) [reply Pod/Classes/WavesLayer.swift]: Will add those back, but I will need some time to resolve the error with Swift3

**Comment by mamdouhweb** [comment]
@jakubknejzlik this should be fixed now with animation

File: Pod/Classes/SonarViewCenteredLayout.swift [hunk 2]
```diff
@@ -53,7 +53,7 @@ extension SonarViewCenteredLayout: SonarViewLayout {
         
         return (Double(index + 1) / Double(numberOfItemsInWave + 1))
     }
-  
+
```
Comment (kafejo) [inline Pod/Classes/SonarViewCenteredLayout.swift]: Please clean up these unnecessary empty lines.

**Review by kafejo — CHANGES_REQUESTED** [review]
Thanks for the pull request! I will merge it and release new version but first please remove the empty spaces that are all over the code. Cheers!

**Comment by mamdouhweb** [comment]
@kafejo that should be done now

**Review by kafejo — APPROVED** [review]
Perfect! Cheers.

**Pull request merged**